cfgov/v1/atomic_elements/organisms.py

```diff
@@ -125,13 +111,8 @@ class Meta:
         template = '_includes/organisms/main-contact-info.html'
 
 
-class SidebarContactInfo(blocks.StructBlock):
-    header = blocks.CharBlock(required=False)
```

Comment:
So, it's not removed just subclassing MainContactInfo which has the same exact fields.